src/locales/de_DE/de_DE.json

```diff
@@ -447,6 +447,7 @@
         "body_record": "Drücken Sie die Taste, wenn Sie bereit sind",
         "button_primary_finish": "Aufnahme beenden",
         "button_primary_next": "Nächster Schritt",
+        "button_primary_start": "Aufnahme starten",
```

Comment:
Should this be added to `MIGRATION.md`?

Reply:
It's already in the migration file (not sure why the key was missing from some of the locales files)

Reply:
I think this was missing because the "Start recording" button is still the icon button in the SDK.